standalone/standalone/src/EggModuleLoader.ts

```
@@ -1,60 +1,63 @@
     }
     return loadUnits;
   }
 
-  static async preLoad(moduleReferences: readonly ModuleReference[]): Promise<void> {
+  static async preLoad(moduleReferences: readonly ModuleReference[], options: EggModuleLoaderOptions): Promise<void> {
     const loadUnits: LoadUnit[] = [];
     const loaderCache = new Map<string, Loader>();
-    const appGraph = EggModuleLoader.generateAppGraph(loaderCache, moduleReferences);
-    appGraph.sort();
-    const moduleConfigList = appGraph.moduleConfigList;
+    const globalGraph = GlobalGraph.instance = EggModuleLoader.generateAppGraph(moduleReferences, options);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)